### ISSUE TITLE
SVTYPE=INS

### ISSUE THREAD

**Issue by zeeev** [hall-lab/svtyper#39]
Hi @cc2qe,

I've split a VCF into many pieces, some are finishing just fine, others contain some kinda of call that is throwing the following error.

It looks INS is genotyping in most cases?  Is this accidental? It runs fine without INS.

 File "/net/eichler/vol8/home/zevk/tools/svtyper/svtyper", line 1519, in <module>
    sys.exit(main())
  File "/net/eichler/vol8/home/zevk/tools/svtyper/svtyper", line 1514, in main
    args.dump)
  File "/net/eichler/vol8/home/zevk/tools/svtyper/svtyper", line 1273, in sv_genotype
    if o1_is_reverse: posA += 1
UnboundLocalError: local variable 'o1_is_reverse' referenced before assignment
Error in job genotype while creating output file split_calls/split_bh.genotyped.vcf.
RuleException:
CalledProcessError in line 12 of /net/eichler/vol24/projects/structural_variation/nobackups/zevk/wham/genotype/Snakefile:
Command '

**Comment by cc2qe** [comment]
SVtyper doesn't currently support insertion calls. We don't identify them with LUMPY so we have ignored them so far. 

How are you guys annotating INS in the VCF? It's not quite clear to me the best way to enable insertion genotyping, but it would require knowledge of the insertion length or its sequence or both.

For now I've just modified svtyper to skip VCF lines with unknown variant types so it doesn't crash (https://github.com/hall-lab/svtyper/commit/eb950034061e4996314a4d1fd144dbbdefb0fe7d)
